deploy/standard/manifests/controller/helm/retina/values.yaml

```diff
@@ -98,6 +98,7 @@ volumeMounts:
   cgroup: /sys/fs/cgroup
   tmp: /tmp
   config: /retina/config
+  host-os-release: /etc/os-release
```

Comment:
I have come across distros where this file is missing since this is open source we should be aware of this. I don't believe Linux states this MUST be present.

Reply:
Right, it's a standard, for not required. Hb this: I remove `Type: File`, and we'll avoid pod start failures if file is missing, k8s will mount and empty dir, which the code handles.

Reply:
@MikeZappa87 Changed this to `FileOrCreate` to avoid pods stuck in Init. Mariner will always have this file non-empty, so this fallback will correctly apply for non-Mariner distros. 
I tested this logic manually.